otter/integration/tests/test_lbsh.py

```diff
@@ -74,25 +74,21 @@ def test_oob_deleted_clb_node(self):
         """
         clb = self.helper.clbs[0]
 
-        nodes = yield clb.list_nodes(self.rcs)
-        self.assertEqual(len(nodes['nodes']), 0,
-                         "There should be no nodes on the CLB yet.")
+        nodes = yield clb.wait_for_nodes(
```

Comment:
Non-blocking - if the nodes aren't needed, it's probably less confusing to just not assign it to a variable, to make it more clear that we are just asserting something about the length, as opposed to doing something with the nodes here.